includes/core.php

```diff
@@ -80,3 +84,73 @@ function maybe_disable_upload_image( $attachment_id ) {
 
 	return $attachment_id;
 }
+
+/**
+ * Adds tweeted status columns for supported post types.
+ */
+function add_tweeted_column() {
+	// Loop through all the supported post types and add tweet status column.
+	$post_types = Utils\get_enabled_post_types();
+	foreach ( (array) $post_types as $post_type ) {
+		add_post_type_support( $post_type, POST_TYPE_SUPPORT_FEATURE );
+		add_filter( "manage_{$post_type}_posts_columns", __NAMESPACE__ . '\modify_post_type_add_tweet_status_column' );
+		add_action( 'manage_' . $post_type . '_posts_custom_column', __NAMESPACE__ . '\modify_post_type_add_tweet_status', 10, 2 );
+	}
+}
+
+/**
+ * Add 'Tweeted' column for supported post types.
+ *
+ * @param array $columns Supported columns for a post type.
+ */
+function modify_post_type_add_tweet_status_column( $columns ) {
+	// Do this so our custom column doesn't end up being the last one, messing up UI.
+	unset( $columns['date'] );
+
+	// Add tweet status column header.
+	$columns['is_tweeted'] = sprintf(
+		'<span class="autoshare-for-twitter-status-logo" title="%s"><span class="screen-reader-text">%s</span></span>',
+		esc_attr__( 'Tweeted status', 'autoshare-for-twitter' ),
+		esc_html__( 'Tweeted status', 'autoshare-for-twitter' )
```

Comment:
You can use `%1$s` here instead.
```suggestion
		'<span class="autoshare-for-twitter-status-logo" title="%2$s"><span class="screen-reader-text">%1$s</span></span>',
		esc_attr__( 'Tweeted status', 'autoshare-for-twitter' )
```

Reply:
One value is used in the attributes while other is inside the tag, so added translation functions, please let me know if that needs change, thanks!